tests/blas_test.cu

```diff
@@ -26,7 +26,6 @@ extern int tdim;
 extern int gridsize_from_cmdline[];
 extern int niter;
 
-extern bool verify_results;
```

Comment:
curious - do we always check now or was it stale? Have only looked at the diff so far.

Reply:
blas_test was changed so that the performance sweep is also a google test, so no verifying the result actually meant nothing is done.  I figured the best thing to do is just get rid of the `verify_results` option here.